projects/1.18/assets/k-turrets/k-turrets/lang/zh_cn.json

```
@@ -0,0 +1,67 @@
+{
+  "itemGroup.k_turrets": "K-Turrets",
+
+  "item.k_turrets.arrow_turret_item": "箭矢炮塔",
+  "item.k_turrets.bullet_turret_item": "火力炮塔",
+  "item.k_turrets.brick_turret_item": "红砖炮塔",
+  "item.k_turrets.firecharge_turret_item": "火焰弹炮塔",
+  "item.k_turrets.steel_ingot": "钢锭",
+  "item.k_turrets.semi_steel": "半成品钢锭",
+  "item.k_turrets.gauss_bullet": "高斯炮弹",
+  "item.k_turrets.gauss_turret_item": "高斯炮塔",
+  "item.k_turrets.cobble_turret_item": "圆石炮塔",
+
+  "item.k_turrets.brick_drone_item": "红砖无人机",
+  "item.k_turrets.bullet_drone_item": "火力无人机",
+  "item.k_turrets.cobble_drone_item": "圆石无人机",
+  "item.k_turrets.arrow_drone_item": "箭矢无人机",
+  "item.k_turrets.gauss_drone_item": "高斯无人机",
+  "item.k_turrets.firecharge_drone_item": "火球无人机",
+
+  "k_turrets.targets": "目标:",
+  "k_turrets.added": "添加到目标",
+  "k_turrets.add.entity.type": "将新的实体添加到目标列表",
+  "k_turrets.choose.tooltip": "按相应按钮选择要删除的实体",
+  "k_turrets.hold.alt.to.scroll": "要滚动目标列表，请按住Alt并使用鼠标滚轮",
+  "k_turrets.dismantle": "拆除",
+  "k_turrets.clear.list": "清除目标列表",
+  "k_turrets.inventory.hint": "潜行并右键单击炮塔/无人机以查看其库存",
+  "k_turrets.claim.turret": "获取此炮塔的权限",
+  "k_turrets.turret.not.yours": "这个炮塔不属于你",
+  "k_turrets.mobile": "可移动",
+  "k_turrets.immobile": "不可移动",
+  "k_turrets.protect.from.players": "射击其他玩家",
+  "k_turrets.not.protect.from.players": "不保护其他玩家",
+  "k_turrets.integrity": "完整",
+  "k_turrets.incorrect.entry": "输入错误",
+  "k_turrets.removed": "已摧毁%1$s个目标",
+  "k_turrets.turret_claimed": "炮塔权限获取成功",
+  "k_turrets.following.owner": "跟随主人",
+  "k_turrets.staying": "停留",
+  "k_turrets.drone_claimed": "无人机权限获取成功",
+  "k_turrets.claim.drone": "获取这架无人机的权限",
+  "k_turrets.drone.not.yours": "这架无人机不属于你",
+  "k_turrets.drone.destroyed": "%1$s已被摧毁",
+  "k_turrets.reset.list": "重置目标",
+  "k_turrets.no.team": "没有友军",
+  "k_turrets.team": "团队",
```

```suggestion
  "k_turrets.no.team": "无团队",
  "k_turrets.team": "团队",
```
也许和原版的scoreboard有点像？